### PR TITLE
Development

### PR DESCRIPTION
**Pull request by pawelKapl**

### CODE REVIEW

File: src/sortingtraining/Strategy.java [hunk 1]
```diff
@@ -32,11 +40,26 @@ SortingStrategy resolve() {
...
+        }
+    },
+    JDK_SORT {
         SortingStrategy resolve() {
             return new DefaultSort();
         }
     },
+    JDK_PARALLEL {
+        SortingStrategy resolve() {
+            return new DefaultSort();
```
Comment (Augustyn) [inline src/sortingtraining/Strategy.java:55]: Shouldn't here be `DefaultParallelSort` ?

File: src/sortingtraining/utils/ComparingUtil.java [hunk 2]
```diff
@@ -16,17 +20,18 @@
...
-
-        times.add(new TimeMeasurement(TimerUtil.measure(new DefaultSort(), numbers), "Default Library Sort"));
+        List<TimeMeasurement> times = Arrays.asList(
+                new TimeMeasurement(TimerUtil.measure(new InsertionSort(), numbers), "Insertion Sort"),
+                new TimeMeasurement(TimerUtil.measure(new ShellSort(), numbers), "Shell Sort"),
+                new TimeMeasurement(TimerUtil.measure(new BubbleSort(), numbers), "Bubble Sort"),
+                new TimeMeasurement(TimerUtil.measure(new SelectionSort(), numbers), "Selection Sort"),
+                new TimeMeasurement(TimerUtil.measure(new MergeSort(), numbers), "Merge Sort"),
+                new TimeMeasurement(TimerUtil.measure(new QuickSort(), numbers), "Quick Sort"),
+                new TimeMeasurement(TimerUtil.measure(new BucketSort(), numbers), "Bucket Sort"),
```
Comment (Augustyn) [inline src/sortingtraining/utils/ComparingUtil.java:30]: Did you consider adding to enum another variable: String, with human readable Sorting alghoritm name?

File: src/sortingtraining/Strategy.java (hunk 1 above)
Comment (pawelKapl) [reply src/sortingtraining/Strategy.java:55]: Of course it should be parallel sort, thx for detecting this bug!

File: src/sortingtraining/utils/ComparingUtil.java (hunk 2 above)
Comment (pawelKapl) [reply src/sortingtraining/utils/ComparingUtil.java:30]: Great idea! Now when i know how to use streams i was also able to refactor this boilerplate code with stream! Check it out: 1633d00

**Pull request merged**